### DISCUSSION TITLE
Thankfull and tips of comments

### DISCUSSION THREAD

**Discussion by Leonidas-Hashakimana** [lsc-hubs/rwanda-hub#20] [General]
Thank you very much for this relevant hub for providing soil and crop datasets to the world's community researchers, policymakers, and academicians as well as farm practitioners. It was keen for so long for Rwandans to ease planning for agriculture development! 

However, for farmers, there is some information that is keen for them such as the quantity of fertilizers recommended to be applied on a specific crop in a specific location [as you select a location]. Then, will this portal function both online and offline? If it is only for online, it will not be as fruitful as it is designed. I can suggest availing offline options for Rwandan farmers (if possible) in order to cut off all usage-hindering factors (e.g., internet). I have not yet been reaching to soil moisture variations datasets. Will they be validated and projected following climate change scenarios ?

Thank you in advance

**Comment by pvgenuchten** [comment]
Hi @Leonidas-Hashakimana, thank you for the suggestions, can I ask you to create a single discussion for each of the topics, so we can close it when it done.

with the current prototype we currently mostly aim to provide useful data for app builders which create decision support apps for farmers, I fully support the suggested scenario of offline use of such a app.

various projects exist which predict soil moisture over time, such as wapor, future-agricultures.org, futurewater.nl. Your impression is that this topic is underrepresented in the catalogue?
